src/amo/components/GetFirefoxButton/index.js

```diff
@@ -89,6 +89,16 @@ export const getDownloadCampaign = ({
   return campaign;
 };
 
+export const getDownloadCategory = (variant?: string): string => {
```

Comment:
Isn't it risky to have an optional parameter? (arity isn't fixed) Could it be not optional?
```suggestion
export const getDownloadCategory = (variant: string): string => {
```

Reply:
Yeah, it doesn't need to be optional. I'll change that.